stdlib/public/TensorFlow/TensorGroup.swift

```diff
@@ -49,3 +51,82 @@ public extension TensorGroup {
     return Array(repeating: nil, count: Int(_tensorHandleCount))
   }
 }
+
+//===----------------------------------------------------------------------===//
+// Conform standard TensorFlow types to TensorGroup
+//===----------------------------------------------------------------------===//
+
+extension TensorHandle : TensorGroup where Scalar : AccelerableByTensorFlow {
```

Comment:
```suggestion
extension TensorHandle : TensorGroup {
```
I see no reason to add such a constraint. `Scalar` is already `AccelerableByTensorFlow` where it's bound.

Reply:
done